### PR TITLE
Added remaining classes of core.display and core.math package

### PR DESCRIPTION
**Pull request by Tiago-Ling**
Kept some children members when documentation was different from parent.

### CODE REVIEW

**Comment by adireddy** [comment]
Nice work 👍  

Your branch is out of date. Can you update and commit.

File: src/pixi/core/math/ObservablePoint.hx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	 * An observable point is a point that triggers a callback when the point's position is changed.
+	 *
+	 * @class
+	 * @memberof PIXI
+	 * @param cb {Function} callback when changed
+	 * @param scope {Object} owner of callback
+	 * @param [x=0] {number} position of the point on the x axis
+	 * @param [y=0] {number} position of the point on the y axis
+	 */
+	function new(cb:Dynamic, scope:Dynamic, x:Float, y:Float);
```
Comment (adireddy) [inline src/pixi/core/math/ObservablePoint.hx]: A few tips :)

The parameters `x` and `y` are optional here so you need to make them optional with `?`

Can you also try and avoid using `Dynamic` whereever possible. For example here the callback `cb` type can be `Void -> Void`.

You can check the status of the pull request after creating it where you can see the build state and checkstyle issues via coeclimate.

Comment (Tiago-Ling) [inline src/pixi/core/math/ObservablePoint.hx]: Ok, just noticed the optionals now! :)

About the Dynamic, i thought about putting Void->Void but wouldn't disallowing parameters reduce the options for the callback too much?

I'm gonna commit the fixes, thanks!

**Comment by Tiago-Ling** [comment]
I think it's ok now - there's nothing that can be done about javascript `Object` other than typing it as `Dynamic` right?

**Comment by adireddy** [comment]
Normally If I know what object contains, I will create a typedef for it. If not it has to be Dynamic.

**Pull request merged**